apps/src/templates/curriculumCatalog/CurriculumCatalog.jsx

```diff
@@ -0,0 +1,37 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {curriculaDataShape} from './curriculumCatalogShapes';
+import i18n from '@cdo/locale';
+import '../../../style/code-studio/curriculum_catalog_container.scss';
+import HeaderBanner from '../HeaderBanner';
+import CourseCatalogIllustration01 from '../../../static/curriculum_catalog/course-catalog-illustration-01.png';
+
+class CurriculumCatalog extends Component {
+  static propTypes = {
+    curriculaData: PropTypes.arrayOf(curriculaDataShape)
+  };
+
+  render() {
+    return (
+      <>
+        <div className={'catalogHeader'}>
+          <HeaderBanner
+            headingText={i18n.curriculumCatalogHeaderTitle()}
+            subHeadingText={i18n.curriculumCatalogHeaderSubtitle()}
+            short={false}
+            imageUrl={CourseCatalogIllustration01}
+          />
+        </div>
+        <div className={'catalogContentContainer'}>
+          <div className={'catalogContent'}>
```

Comment:
I am sorry Turner but I still have concerns with this. I think you did a good job of following the example you were looking at, but I have concerns with that example too 😬 here's what I see in the DOM when I run your code locally: 
![Screenshot 2023-03-28 at 3 46 25 PM](https://user-images.githubusercontent.com/8001765/228384756-df99a2b3-b8aa-4c62-a85c-b293d33c15d0.png)

![Screenshot 2023-03-28 at 3 47 18 PM](https://user-images.githubusercontent.com/8001765/228384769-3c99845e-0367-4fe6-99c9-06f5588fdff4.png)

the problem is that these classnames are not scoped to your code, and the css is put inline into the DOM, meaning anyone who used the same classname would accidentally get your styles. it _probably_ won't break anything but this is not our best practice. 

here is an example using CSS modules:
* https://github.com/code-dot-org/code-dot-org/blob/bb17e9b972b13d4bb1e04f5bf6eef4422a277200/apps/src/applab/PhoneFrame.jsx#L4
* https://github.com/code-dot-org/code-dot-org/blob/bb17e9b972b13d4bb1e04f5bf6eef4422a277200/apps/src/applab/PhoneFrame.jsx#L29-L36

now in the DOM you can see that the classnames are obfuscated:
![Screenshot 2023-03-28 at 3 52 06 PM](https://user-images.githubusercontent.com/8001765/228385367-93cbc5ea-a71a-40af-8ff8-7274f2c8cd25.png)

Please take a crack at switching to CSS modules. it shouldn't be much work to change, just a different example to follow.

CC @megcrenshaw since it looks like curriculum_catalog_card.scss has the same issue.

Reply:
Ohh gotcha gotcha, on it! Thank you so much for the detailed feedback, I understand now!